docs/content/latest/releases/whats-new/_index.md

```diff
@@ -41,58 +40,38 @@ menu:
+- [DocDB] Improve row scanning by using SeekForward for intents. In testing, performance of `SELECT COUNT(*)` has improved by 66%. [#4277](https://github.com/yugabyte/yugabyte-db/issues/4277)
+- [DocDB] Add asynchronous transaction status resolution to conflict detection. [#4058](https://github.com/yugabyte/yugabyte-db/issues/4058)
 
 ## Yugabyte Platform
 
-- When shrinking a universe, remove nodes in descending index order. [#3292](https://github.com/yugabyte/yugabyte-db/issues/3292)
-- Add back up and restore of Yugabyte Platform using `yb_platform_backup.sh` script. [#4208](https://github.com/yugabyte/yugabyte-db/issues/4208)
-- Change `yb_backup.py` to use the `yb-admin` changes for backup-related changes for snapshot (see [System improvements](#system-improvements) above).
-- Fix expected restoration state in the `yb_backup.py` script.
-- Allow users to select multiple single tables to backup in addition to specifying a full universe backup. [#3680](https://github.com/yugabyte/yugabyte-db/issues/3680)
-- Customize the SMTP server for sending alert messages using configuration entries for `smtpData` (`smtpServer`, `smtpPort`, `emailFrom`, `smtpUsername`, `smtpPassword`, `useSSL`, and `useTLS`). [#4201](https://github.com/yugabyte/yugabyte-db/issues/4201)
-- [YW] Add option to specify table keyspace when creating manual or scheduled backups. [#3342](https://github.com/yugabyte/yugabyte-db/issues/3342)
-- For Azure Storage blob backups, use SAS tokens instead of Service Principal client secrets.
-- Add create and restore backup support for Azure Blob Storage with SAS tokens. [#3721](https://github.com/yugabyte/yugabyte-db/issues/3721)
-- [YW] Add **IAM Role** toggle in provider storage configuration to use the IAM role instead of requiring an **Access Key** and **Secret**. [#4204](https://github.com/yugabyte/yugabyte-db/issues/4204)
-- [YW] When creating a universe and AWS provider is selected, display new **Use IAM Profile** toggle and **ARN String** text field. [#4199](https://github.com/yugabyte/yugabyte-db/issues/4199)
-- Use Raft configuration as a source for master addresses in `server.conf` for master. [#4089](https://github.com/yugabyte/yugabyte-db/issues/4089)
-- [YW] If a node appears as unreachable, it can be removed or released without generating errors. [#4171](https://github.com/yugabyte/yugabyte-db/issues/4171)
-- [YW] Create GCP providers with any combination of host credentials and host (or shared) VPC. [#4177](https://github.com/yugabyte/yugabyte-db/issues/4177)
+- When performing a full move or add node on a universe that has a yb-master, the `server.conf` file is now being updated with the new `master_addresses`. [#4242](https://github.com/yugabyte/yugabyte-db/issues/4242)
+
+### YugabyteDB Admin Console
```

Comment:
this section header should be removed and all these things should be listed only under Yugabyte Platform.

Reply:
Fixed